pkg/vectorize/asin/asin.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
```

Comment:
move files related to asin to plan2/function/builtin/unary. The vectorize directory may be deleted in future versions.